src/app.js

```diff
@@ -276,11 +276,16 @@ export function app(app) {
           removeElement(element, reusableChild[0], reusableNode)
         }
       }
-    } else if (node !== oldNode) {
+    } else if (
+      element != null &&
+      node !== oldNode &&
+      node !== element.nodeValue
+    ) {
       var i = element
       parent.replaceChild((element = createElement(node)), i)
```

Comment:
Sure! I'm happy to answer all your questions and please do find all its secrets and help us make it better! 😅 

The number to string conversion is right [here](https://github.com/hyperapp/hyperapp/blob/master/src/h.js#L16).